src/chai-form.ts

```diff
@@ -341,15 +358,17 @@ export class ChaiForm extends LitElement {
   }
 
   override render() {
+debugger
     return html`
       <h2>${this.headerText}</h2>
       <slot name="before"></slot>
       <form id="chai-quote-form">
         <slot>
-          <chai-name></chai-name>
-          <chai-phone></chai-phone>
-          <chai-email></chai-email>
-          <chai-address></chai-address>
+
+          ${this.isCreateName ?  html`<chai-name id="chai-name"></chai-name>` : ''}
```

Comment:
Please explain what this is doing with an inline comment.

Reply:
I would have liked to see this work correctly without any changes to the code inside `chai-form.ts`, because I'm concerned about breaking existing users. Would it be possible to add/remove the `<chai-name>` etc. elements using code in _`chai-form-configurator.ts`_ instead?